### DISCUSSION TITLE
TypeError: shared.incrementer is not a function

### DISCUSSION THREAD

**Discussion by labbedaine** [nuxt-modules/i18n#2627] [Q&A] [answered]
Hello!

I upgraded the dependency from **^8.0.0-rc.3** to **^8.0.0-rc.11** and I am now getting the following error.

<img width="870" alt="image" src="https://github.com/nuxt-modules/i18n/assets/1533999/efa30341-6960-4ab3-8347-91c8f4a43ba4">

Any suggestion?
Thanks

**Comment by BobbieGoede** [comment]
This sounds like a dependency issue, can you share your `package.json`? Also have you tried deleting `node_modules` and lockfiles and running install again? This should resolve issues caused by dependency updates between the releases.

**Reply by labbedaine** [reply]
Hey! Thanks for the reply

So, yes, I've deleted **node_modules** and **package-lock.json** but same problem.  Here's a copy of my package.json

> {
>   "name": "nuxt-app",
>   "version": "0.1.0",
>   "private": true,
>   "scripts": {
>     "build": "nuxt build",
>     "dev": "nuxt dev",
>     "generate": "nuxt generate",
>     "preview": "nuxt preview",
>     "postinstall": "nuxt prepare",
>     "release:start": "gitex-flow release start",
>     "release:finish": "gitex-flow release finish",
>     "hotfix:start": "gitex-flow hotfix start",
>     "hotfix:finish": "gitex-flow hotfix finish",
>     "test": "vitest",
>     "ci:test": "vitest run --reporter=junit --outputFile.junit=./test-reports/results.xml",
>     "coverage": "vitest run --coverage",
>     "lint:js": "eslint --ext \".ts,.js,.vue\" --ignore-path .gitignore .",
>     "lint:prettier": "prettier --check .",
>     "lint": "npm run lint:js && npm run lint:prettier",
>     "lintfix": "prettier --write --list-different . && npm run lint:js --fix",
>     "migrate:up": "cd ./server && npx sequelize-cli db:migrate",
>     "migrate:seed": "cd ./server && npx sequelize-cli db:seed:all"
>   },
>   "devDependencies": {
>     "@nuxt/devtools": "latest",
>     "@nuxt/test-utils": "^3.8.0",
>     "@nuxtjs/eslint-config-typescript": "^12.1.0",
>     "@nuxtjs/eslint-module": "^4.1.0",
>     "@nuxtjs/i18n": "^8.0.0-rc.3",
>     "@pinia-plugin-persistedstate/nuxt": "^1.2.0",
>     "@pinia/testing": "^0.1.3",
>     "@playwright/test": "^1.39.0",
>     "@types/node": "^18",
>     "@types/pg": "^8.10.7",
>     "@typescript-eslint/eslint-plugin": "^6.9.1",
>     "@vitest/coverage-istanbul": "^0.33.0",
>     "@vitest/coverage-v8": "^0.33.0",
>     "@vue/test-utils": "^2.4.1",
>     "eslint": "^8.52.0",
>     "eslint-config-prettier": "^9.0.0",
>     "eslint-plugin-prettier": "^5.0.1",
>     "eslint-plugin-vue": "^9.18.1",
>     "gitex-flow": "^2.4.0",
>     "happy-dom": "^10.11.2",
>     "import": "^0.0.6",
>     "sequelize-cli": "^6.6.1",
>     "nuxt": "^3.8.0",
>     "nuxt-typed-router": "^3.3.2",
>     "nuxt-vitest": "^0.11.0",
>     "prettier": "^3.0.3",
>     "typescript": "^5.2.2",
>     "vite-tsconfig-paths": "^4.2.1",
>     "vitest": "^0.33.0",
>     "vuetify-nuxt-module": "^0.6.0"
>   },
>   "dependencies": {
>     "@pinia/nuxt": "^0.5.1",
>     "@vuelidate/core": "^2.0.3",
>     "@vuelidate/validators": "^2.0.4",
>     "aws-amplify": "^5.3.11",
>     "pg": "^8.11.3",
>     "pg-hstore": "^2.3.4",
>     "pinia": "^2.1.7",
>     "sequelize": "^6.33.0",
>     "sequelize-cli": "^6.6.1"
>   },
>   "overrides": {
>     "@intlify/core-base": "9.3.0-beta.25",
>     "@intlify/shared": "9.3.0-beta.25",
>     "vue": "latest"
>   }
> }

**Reply by labbedaine** [reply, marked as answer]
I maybe found the culprit...

```
"@intlify/core-base": "9.3.0-beta.25",
"@intlify/shared": "9.3.0-beta.25",
```
